src/components/Header/index.js

```diff
@@ -2,10 +2,19 @@ import Link from 'next/link';
 import {useRouter} from 'next/router';
 
 import SVG from '../../../public/SVG/svg';
+import useStore from '../../hooks/useStore';
+import StyledCartCounter from '../CartCounter/styled';
+import StyledLink from '../Link/styled';
 
 import StyledHeader from './styled';
 
 export default function Header() {
+	const products = useStore(state => state.products);
+	const initialValue = 0;
```

Comment:
Just a nitty note - the name initialValue variable holds your initial quantity.  🤓
```suggestion
	const initialQuantity = 0;
```